backend/app/views/spree/admin/images/index.html.erb

```diff
@@ -13,6 +13,7 @@
   <%= render 'new', product: @product, image: Spree::Image.new(viewable: @product) %>
 </div>
 
+<% if can?(:create, Spree::Image) %>
```

Comment:
The inner content needs an indentation here

Reply:
@kennyadsl . You're right. I'll fix it.